### PR TITLE
Add alUSD

### PR DESCRIPTION
**Pull request by imimim-username**
I've added alUSD per the instructions that were sent. Hopefully, I've done this correctly. Thanks

### CODE REVIEW

File: src/public/CowSwap.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "CoW Swap",
   "timestamp": "2022-12-15T19:00:00+00:00",
   "version": {
-    "major": 1,
-    "minor": 0,
-    "patch": 1
+    "major": 0,
```
Comment (fleupold) [inline src/public/CowSwap.json:5]: This should bump minor version by one from the current version (https://files.cow.fi/tokens/CowSwap.json), to 1.2.0

**Review by fleupold — COMMENTED** [review]
Looks good to me, just needs the version bump.

**Comment by imimim-username** [comment]
> This should bump minor version by one from the current version (
https://files.cow.fi/tokens/CowSwap.json), to 1.2.0

Hi. Thanks for the review and comment. Please bear with my ignorance. Does
your comment mean that I need to change the version bump text that I
edited? Based on what I read, I interpreted it as when a token is added,
"minor" value should be 1. I guess I mis-interpreted how it works. Should I
change it to 1, 2, 0 for major, minor, and patch, respectively?

Many thanks

**Comment by fleupold** [comment]
I merged main and fixed the versioning. Change is deployed

Check it out: 
image

**Comment by imimim-username** [comment]
Thank you so much! I think I'm finally understanding how the versioning
works. Again, thanks for bearing with me and for your help!

**Pull request merged**